gobblin-metastore/src/test/java/org/apache/gobblin/metastore/testing/TestMetastoreDatabaseServer.java

```diff
@@ -77,6 +79,7 @@ class TestMetastoreDatabaseServer implements Closeable {
   private final String dbUserPassword;
   private final String dbHost;
   private final int dbPort;
+  private static Map<String, Injector> injectors = new HashMap<>();
```

Comment:
I don't understand who's using this.  is it basically a per-JDBC-URI mapping that caches the `Injector`, so it's not created multiple times?

if so, should this method either be synchronized or we use `ConcurrentHashMap`?

also, naming could be clearer, such as `injectorsByJdbcUri`